CHANGELOG.md

```diff
@@ -9,18 +9,54 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Fixed
+
+
+### Added
+
+### Changed
+
+
+-->
+## v1.1.0
+### Fixed
 - Autofill for empty slug
+- Fixed Web content preview in CMS
+- Fixed related pages export
+- Fixed API tests
+
 
 ### Added
+- Added support for SMS content
+- Added support for USSD content
+- Added support for a Footer in WhatsApp content
+- Added support for List Messages in WhatsApp content
+- Added error handling on WhatsApp template submission errors, and adds issue to sentry
+- Added validation of variables used in WhatsApp template submission
+- Added WhatsApp title check
+- Added typing information for migration test
+- Added CI checks for pending migrations
+- Fixed Redis caching issues
+
+
 
 ### Changed
 - Moved slug uniqueness validation to the model validation/clean
 - Empty slugs will auto-generate a unique slug, but if a duplicate slug is specified the user will get a validation error instead of their chosen slug getting overwritten with a unique one.
 - Slug uniqueness is per-locale
 - Test speedups. Tests now run in parallel by default, and there's a separate contentrepo/settings/test.py for test-specific settings.
 - Tests no longer run in parallel by default, because the output is a little less clear and the speedup is negligible on some systems.
```

Comment:
We can probably combine these two (tests now run in parallel, tests don't run in parallel anymore).
Perhaps `Test speedups, and there's a separate contentrepo/settings/test.py for test-specific settings.`

Reply:
Thanks, done